### PR TITLE
Add LightZoneProject.LightZone

### PR DESCRIPTION
**Pull request by ktgw0316**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/96448)

### CODE REVIEW

File: manifests/l/LightZoneProject/LightZone/4.2.4/LightZoneProject.LightZone.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LightZoneProject.LightZone
+PackageVersion: 4.2.4
+PackageLocale: en-US
+Publisher: LightZone Project
+PublisherUrl: https://github.com/ktgw0316/LightZone/
```
Comment (mdanish-kh) [inline manifests/l/LightZoneProject/LightZone/4.2.4/LightZoneProject.LightZone.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://github.com/ktgw0316/
```

File: manifests/l/LightZoneProject/LightZone/4.2.4/LightZoneProject.LightZone.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LightZoneProject.LightZone
+PackageVersion: 4.2.4
+PackageLocale: en-US
+Publisher: LightZone Project
+PublisherUrl: https://github.com/ktgw0316/LightZone/
+PackageName: LightZone
+PackageUrl: https://github.com/ktgw0316/LightZone/releases/download/4.2.4/LightZone_Installer_4.2.4_20220227_windows-x64.exe
```
Comment (mdanish-kh) [inline manifests/l/LightZoneProject/LightZone/4.2.4/LightZoneProject.LightZone.locale.en-US.yaml:10]:
```suggestion
PackageUrl: https://github.com/ktgw0316/LightZone
```

File: manifests/l/LightZoneProject/LightZone/4.2.4/LightZoneProject.LightZone.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: LightZoneProject.LightZone
+PackageVersion: 4.2.4
+PackageLocale: en-US
+Publisher: LightZone Project
+PublisherUrl: https://github.com/ktgw0316/LightZone/
+PackageName: LightZone
+PackageUrl: https://github.com/ktgw0316/LightZone/releases/download/4.2.4/LightZone_Installer_4.2.4_20220227_windows-x64.exe
+License: BSD-3-Clause
+ShortDescription: a professional-level digital darkroom and photo editor
```
Comment (mdanish-kh) [inline manifests/l/LightZoneProject/LightZone/4.2.4/LightZoneProject.LightZone.locale.en-US.yaml:12]:
```suggestion
ShortDescription: A professional-level digital darkroom and photo editor.
```

File: manifests/l/LightZoneProject/LightZone/4.2.4/LightZoneProject.LightZone.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LightZoneProject.LightZone
+PackageVersion: 4.2.4
+PackageLocale: en-US
+Publisher: LightZone Project
+PublisherUrl: https://github.com/ktgw0316/LightZone/
+PackageName: LightZone
+PackageUrl: https://github.com/ktgw0316/LightZone/releases/download/4.2.4/LightZone_Installer_4.2.4_20220227_windows-x64.exe
+License: BSD-3-Clause
```
Comment (mdanish-kh) [inline manifests/l/LightZoneProject/LightZone/4.2.4/LightZoneProject.LightZone.locale.en-US.yaml:11]:
```suggestion
License: BSD-3-Clause
LicenseUrl: https://github.com/ktgw0316/LightZone/blob/master/COPYING
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/l/LightZoneProject/LightZone/4.2.4/LightZoneProject.LightZone.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: LightZoneProject.LightZone
+PackageVersion: 4.2.4
```
[the remark this answers is not in the document]
Comment (ktgw0316) [reply manifests/l/LightZoneProject/LightZone/4.2.4/LightZoneProject.LightZone.installer.yaml:5]: > If you're the developer of the package, I would highly recommend writing a DisplayVersion String to Registry

Thank you for the great suggestion!  I'll add it in next release of the package.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**